models/agent.ts

```diff
@@ -0,0 +1,23 @@
+import dbClient from "../lib/db";
+
+export async function createAgent(agent) {
+  const id = await dbClient.agent.create({data: agent, select: id});
```

Comment:
> `select: id`, 

where does `id` come from?

Reply:
`id` is an internal-to-postgres field that Prisma requires in order to represent the AgentProfile-to-Agent relation
https://www.prisma.io/docs/concepts/components/prisma-schema/relations#the-relation-attribute
you can see how it connects in `schema.prisma`
```
model AgentProfile {
  ...
  agent      Agent       @relation(references: [id], fields: [agentId])
  agentId    String
...
}

model Agent {
  id                         String @default(cuid())
....
  agentProfiles      AgentProfile[]
}
```